internal/service/employee/employee_default.go

```diff
@@ -27,8 +27,8 @@ func (s *service) GetAll() ([]models.Employee, error) {
 }
 
 func (s *service) GetById(id int) (models.Employee, error) {
-	employee, ok := s.repository.GetById(id)
-	if !ok {
+	employee, err := s.repository.GetById(id)
+	if err != nil {
 		newError := fmt.Errorf("%w : employee with id %d not found", errors.ErrNotFound, id)
```

Comment:
La validación de si es un error, o un not found, ya se realizó desde el repository

Al handler le llega el error y ya la función handleError mira cual error es

Desde el handler solo se deberia retornar el error
```suggestion
	if err != nil {
		return models.Seller{}, err
	}
```